Server/DeviceManagement/DeviceManagement.Application/Dto/AreaInputDto.cs

```diff
@@ -0,0 +1,5 @@
+﻿using System;
+
+namespace ModularHouse.Server.DeviceManagement.Application.Dto;
+
+public record AreaInputDto(string Name, string Description, Guid UserId);
```

Comment:
Future InputDtos can be larger, so it's better to make entities' related dtos by using `{ get; init; }` keywords:
```
public record AreaInputDto
{
    string Name { get; init; }
    string Description { get; init; }
    Guid UserId { get; init; }
}
```